metals/src/main/scala/scala/meta/internal/metals/formatting/MultilineString.scala

```diff
@@ -100,9 +134,9 @@ case class MultilineString(userConfig: () => UserConfiguration)
   }
 
   private def hasNQuotes(start: Int, text: String, n: Int): Boolean =
-    (start until start + n).forall(i =>
+    (start until start + n).forall { i =>
       if (i < 0 || i > text.length) false else text(i) == quote
```

Comment:
Not related to this change, but I encounter some `IndexOutofBoundsException` when I write quote in the end of a file, maybe we should check the length of text before accessing `text(I)` :thinking:

Reply:
hmmm, it seems that it is checking `i > text.length`

Reply:
Ah, I got it, if i == text.length it will fail

Reply:
it should be i >= text.length

Reply:
I wonder why it is if `(i < 0 || i > text.length)` in the first place, it is not checking the `0` index either.